### PR TITLE
[inductor] remove `fft` and `svd` ops from `fake_incorrect_kernels`

### PR DESCRIPTION
**Pull request by nkaretnikov**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #103616

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78

### CODE REVIEW

**Review by eellison — APPROVED** [review]
LGTM if CI Passes

**Comment by nkaretnikov** [comment]
@eellison would failures in `fake_tensor_stride_failing_ops` be a blocker for merging this if that fails?

**Comment by eellison** [comment]
Yea.. If there are striding failures then I think we shouldn't land this. Otherwise we'll get runtime assert failures in inductor

**Comment by nkaretnikov** [comment]
@eellison Please take another look: I've pushed an update implementing stride logic from `_exec_fft` in `meta_fft_c2c`.

@XiaobingSuper Could you test if this PR fixes the issues you were having with GoogleFnet? This PR now runs the repro from https://github.com/pytorch/pytorch/pull/88248#discussion_r1223842216 instead of erroring out (note: the trailing paren is missing in the repro).  

```py
Using FallbackKernel: torch.ops.prims.convert_element_type.default
Using FallbackKernel: aten._fft_c2c.default
tensor([[[  740.4529+0.0000j,   126.5869-135.1518j, -1001.8738-182.1360j,
...
```

**Review by eellison — APPROVED** [review]
😍 😍 😍

File: torch/_meta_registrations.py
Comment (peterbell10) [inline torch/_meta_registrations.py:192]: I believe cpu and cuda results diverge in some cases because cuFFT can only transform up to 3 dimensions at once, so we end up taking multiple passes for higher dimensions. This is still better than assuming contiguous tough.

**Comment by XiaobingSuper** [comment]
> @eellison Please take another look: I've pushed an update implementing stride logic from `_exec_fft` in `meta_fft_c2c`.
> 
> @XiaobingSuper Could you test if this PR fixes the issues you were having with GoogleFnet? This PR now runs the repro from [#88248 (comment)](https://github.com/pytorch/pytorch/pull/88248#discussion_r1223842216) instead of erroring out (note: the trailing paren is missing in the repro).
> 
> ```python
> Using FallbackKernel: torch.ops.prims.convert_element_type.default
> Using FallbackKernel: aten._fft_c2c.default
> tensor([[[  740.4529+0.0000j,   126.5869-135.1518j, -1001.8738-182.1360j,
> ...
> ```

@nkaretnikov, thanks, I checked it on my side, and it works now.

**Pull request closed without merging**
